spharpy/samplings/coordinates.py

```diff
@@ -36,7 +36,17 @@ def __init__(
         sh_order : int, optional
             Maximum spherical harmonic order of the sampling grid.
             The default is ``None``.
+        radius_tolerance : float, optional
+            All points that are stored in a SamplingSphere must have the same
+            radius and an error is raised if the maximum deviation from the
+            mean radius exceeds this tolerance. The default of ``1e-6`` meter
+            is intended to allow for some numerical inaccuracy.
         """
+        # must be initialized first, because it is already required for
+        # checking if input points have the same radius
+        self._radius_tolerance = None
```

Comment:
```suggestion
```
is this line required?

Reply:
Yes otherwise the setter fails

Reply:
is this still the case? I cant find a reason, why this should happen.
I think we should avoid setting the private variable out of the setter.